### ISSUE TITLE
Not all Pokémon

### ISSUE THREAD

**Issue by GrafOnega** [AHAAAAAAA/PokemonGo-Map#2851]
My comma is
python runserver.py -a google -u *** -p *** -st 12 -k *** -l "Oktyabrskaya,NizhnyNovgorod,Russia" -t 3 -sd 5 -wh http://127.0.0.1:4000

It's all Pokémon who found. Obviously, much more of them after the fact.
https://imgur.com/a/ETCmr

**Comment by tarthim** [comment]
Run -t 1, more threads is borked up at the moment.

**Comment by GrafOnega** [comment]
@tarthim 
Nothing changes. Like.

**Comment by jangrewe** [comment]
-t 2 and -t3 was working quite well for me, but right now i do see the same issue.

**Comment by lepschay** [comment]
same here. tested with latest dev and stable version. maybe niantic has changed something to prevent us scanning rare pokémon?

**Comment by GrafOnega** [comment]
Maybe. Yesterday rare Pokemon spawn periodically. Today, no one.

**Comment by jangrewe** [comment]
Maybe they lowered the radius again, to even less than 70m? Didn't somebody have an API call to get their actual distance yesterday?

**Comment by externuz** [comment]
We try to use 30m radius and request delay = 5s, but id does not helps. Sorry

**Comment by GrafOnega** [comment]
@externuz Дай, пожалуйста, свой ВК :)

**Comment by externuz** [comment]
vk.com/externuz

**Comment by SpechtXXO** [comment]
Some new pokemon which appear on the map have a remaining time of about 4 minutes. It seems like the were discovered after the second or third scan. Can someone confirm that?

**Comment by kayzonami** [comment]
Yeah I was using this all day yesterday and noticed halfway throughout the day that I missed some Pokemon with my scans (ie. a Snorlax well inside my scan area), or some showed up on my map too late (<5 mins left). Was using -st 5 -t 2 -sd 2 since I got no errors, but it seems like even with no errors Pokemon are still missed somehow? Will try -sd 5 today and see if anything changes.

**Comment by MysticalOS** [comment]
I def noticed that anything less than 5 starts to just not find nearly as many. when i run two side by side and compare faster to slower, slower finds more. So i suspect some throttling is at play.
